ProjectSettings/ProjectSettings.asset

```diff
@@ -54,7 +54,7 @@ PlayerSettings:
   androidShowActivityIndicatorOnLoading: -1
   tizenShowActivityIndicatorOnLoading: -1
   iosAppInBackgroundBehavior: 0
-  displayResolutionDialog: 1
+  displayResolutionDialog: 2
```

Comment:
What does this do?

Reply:
Changes "[Display Resolution Dialog](https://docs.unity3d.com/Manual/class-PlayerSettingsStandalone.html)" from "Enabled" to "Hidden By Default".

![image](https://user-images.githubusercontent.com/3580640/43553370-7ca583fa-95a3-11e8-8c01-85dd9bb02a72.png)

This is the popup that shows up after you build for Standalone and launch the exe. With this change, that popup only appears if you launch the app while holding alt. I've found that it helps for ease of launching after building, especially when a headset is involved.